dashboard/test/ui/features/header.feature

```diff
@@ -40,41 +40,38 @@ Scenario: Teacher in English should see 5 header links
   And I see "#header-teacher-professional-learning"
   And element "#header-teacher-professional-learning" contains text "Professional Learning"
 
-@skip
 Scenario: Signed out user in Spanish should see 2 header links
   Given I am on "http://code.org/lang/es"
   Then check that I am on "http://code.org/"
   And I dismiss the language selector
   Given I am on "http://studio.code.org/courses"
   And I wait to see ".headerlinks"
   And I see "#header-non-en-courses"
-  And element "#header-non-en-courses" has "es" text from key "nav.header.courses"
+  And element "#header-non-en-courses" has "es" text from key "nav.header.course_catalog"
   And I see "#header-non-en-projects"
   And element "#header-non-en-projects" has "es" text from key "nav.header.project_gallery"
 
-@skip
 Scenario: Student in Spanish should see 2 header links
   Given I create a student named "Estrella Estudiante"
-  Then check that I am on "http://studio.code.org/courses"
+  Then check that I am on "http://studio.code.org/home"
   Given I am on "http://studio.code.org/courses/lang/es"
   Then check that I am on "http://studio.code.org/courses"
   And I wait to see ".headerlinks"
   And I see "#header-student-courses"
-  And element "#header-student-courses" has "es" text from key "nav.header.courses"
+  And element "#header-student-courses" has "es" text from key "nav.header.course_catalog"
   And I see "#header-student-projects"
   And element "#header-student-projects" has "es" text from key "nav.header.project_gallery"
 
-@skip
 Scenario: Teacher in Spanish should see 5 header links
   Given I create a teacher named "Pabla Profesora"
   Then check that I am on "http://studio.code.org/home"
   Given I am on "http://studio.code.org/home/lang/es"
   Then check that I am on "http://studio.code.org/home"
   And I wait to see ".headerlinks"
   And I see "#header-teacher-home"
-  And element "#header-teacher-home" has "es" text from key "nav.header.home"
+  And element "#header-teacher-home" has "es" text from key "nav.header.my_dashboard"
   And I see "#header-teacher-courses"
-  And element "#header-teacher-courses" has "es" text from key "nav.header.courses"
```

Comment:
I don't think we use `nav.header.home` or `nav.header.courses` anywhere. Am I missing something? If not, I remove from https://github.com/code-dot-org/code-dot-org/blob/47379314467b1bad950908600ef9473b893dd8e7/dashboard/config/locales/en.yml in a follow up.